### ISSUE TITLE
Cannot toggle upvote

### ISSUE THREAD

**Issue by laymonage** [giscus/giscus#31]
I've already done this in #18 but for some reason GitHub throws `FORBIDDEN` errors. Maybe we need to ping some hubbers and see what's the issue?

**Comment by laymonage** [comment]
Upstream discussion: https://github.com/github/feedback/discussions/3968

**Comment by laymonage** [comment]
Update: they're aware of this problem and have put this into their to-do list.

**Comment by alejandrohdezma** [comment]
I noticed Giscus adds an upvote to any user comment, is it related to this issue?

**Comment by laymonage** [comment]
@alejandrohdezma yeah it's already implemented, but it doesn't work until GitHub allows toggling upvotes using the API with an app-issued token.

**Comment by alejandrohdezma** [comment]
Good to know! Thanks! 😸

**Comment by git-Hmmm** [comment]
@laymonage Can you please hide .gsc-upvote-button gsc-social-reaction-summary-item with css until then it starts working, I am telling this just because it doesn't looks good when visitors see something is broken is there any way I can inject css for me only into that iframe?

**Comment by LukePrior** [comment]
Yeah having an option to toggle a simple CSS rule that set `display: none` on `gsc-upvote-button` would be great.

**Comment by SeryiBaran** [comment]
> Yeah having an option to toggle a simple CSS rule that set `display: none` on `gsc-upvote-button` would be great.

I made a couple of themes that add only these styles:
https://github.com/SeryiBaran/seryibaran.github.io/tree/main/assets/GSThemes

**Comment by TracyYXChen** [comment]
@laymonage is it possible to hide the upvote button since it does not function properly? I tested the script manually on [my page](https://tracyyxchen.github.io/blog/2023/about/) in the console but it can't hide it. Thanks in advance!
```
let elements = document.getElementsByClassName('gsc-upvote-button');
for(let i=0; i < elements.length; i++){
                        elements[i].style.display = 'none';
 }
```

**Comment by laymonage** [comment]
Given that it's been two years and GitHub still hasn't fixed this, I've opted to disable the button and change the styling a bit. I don't hide it completely as I think it's still useful to see the number of upvotes on the comment.

**Comment by ouuan** [comment]
> I don't hide it completely as I think it's still useful to see the number of upvotes on the comment.

Maybe there could be an option to hide it?

**Comment by laymonage** [comment]
Sorry, I don't like to add an option that would likely be removed in the future. If you really want to hide it, you can use a custom theme or self-host giscus.
